src/ui/ShareQuery/ShareQuery.ts

```diff
@@ -147,21 +147,37 @@ export class ShareQuery extends Component {
     return content;
   }
 
+  private buildCompleteQuery() {
+    this.completeQuery = <HTMLInputElement>$$('input', {
+      type: 'text',
+      className: 'coveo-share-query-summary-info-input'
+    }).el;
+
+    const lastQuery = this.queryController.getLastQuery();
+    this.completeQuery.value = Utils.trim(this.outputIfNotNull(lastQuery.q) + ' ' + this.outputIfNotNull(lastQuery.aq) + ' ' + this.outputIfNotNull(lastQuery.cq));
```

Comment:
`Utils.trim(_.filter([lastQuery.q, lastQuery.aq, lastQuery.cq], (queryPart) => this.outputIfNotNull(queryPart)).join(' '));` ?